docs/devops-guide/docker.md

```diff
@@ -26,17 +26,19 @@ follow these steps:
 4. Create required `CONFIG` directories
    * For linux: 
    ```bash
-   mkdir -p ~/.jitsi-meet-cfg/{web/crontabs,web/letsencrypt,transcripts,prosody/config,prosody/prosody-plugins-custom,jicofo,jvb,jigasi,jibri}
+   mkdir -p ~/.jitsi-meet-cfg/{web,transcripts,prosody/config,prosody/prosody-plugins-custom,jicofo,jvb,jigasi,jibri}
```

Comment:
Are you sure those are not needed? I never needed them, but we added them because some people had run into issues.

Reply:
okay, I tried multiple times and they seemed superfluous, especially since there was never any file created in the `letsencrypt` folder, seems to be all in `acme` now

Reply:
Oh, that's right 🤦